processing/src/main/java/io/druid/segment/filter/AndFilter.java

```diff
@@ -149,6 +150,17 @@ public boolean supportsBitmapIndex(BitmapIndexSelector selector)
   }
 
   @Override
+  public double estimateSelectivity(ColumnSelector columnSelector, BitmapIndexSelector indexSelector)
+  {
+    // Estimate selectivity with attribute value independence assumption
```

Comment:
My concern is that for somebody this assumption could be wrong, and some selectivity computed incorrectly, and the overall select query chooses suboptimal strategy and performs worse.

Ultimately in this PR, I want to ensure that there won't be performance regression